frb_example/pure_dart/rust/src/api.rs

```diff
@@ -958,36 +962,56 @@ pub fn nested_id(id: [TestId; 4]) -> [TestId; 2] {
     }
 }
 
-/// Opaque types
+pub fn sync_dart_opaque(not_temp: DartOpaque) -> SyncReturn<String> {
+    SyncReturn("test".to_owned())
+}
+
+pub fn async_dart_opaque(not_temp: DartOpaque) -> String {
```

Comment:
uber-nit: What about `sync_accept_dart_opaque` etc, i.e. tell readers we are using it as arg